apps/meteor/tests/unit/server/lib/dataExport/exportRoomMessagesToFile.spec.ts

```diff
@@ -0,0 +1,213 @@
+import { expect } from 'chai';
+import { before, describe, it } from 'mocha';
+import proxyquire from 'proxyquire';
+import sinon from 'sinon';
+
+// Create stubs for dependencies
+const stubs = {
+	findPaginatedMessages: sinon.stub(),
+	mkdir: sinon.stub(),
+	writeFile: sinon.stub(),
+	findPaginatedMessagesCursor: sinon.stub(),
+	findPaginatedMessagesTotal: sinon.stub(),
+	translateKey: sinon.stub(),
+	settings: sinon.stub(),
+};
+
+const { exportRoomMessages, exportMessageObject } = proxyquire
+	.noCallThru()
+	.load('../../../../../server/lib/dataExport/exportRoomMessagesToFile.ts', {
+		'@rocket.chat/models': {
+			Messages: {
+				findPaginated: stubs.findPaginatedMessages,
+			},
+		},
+		'fs/promises': {
+			mkdir: stubs.mkdir,
+			writeFile: stubs.writeFile,
+		},
+		'../i18n': {
+			i18n: {
+				t: stubs.translateKey,
+			},
+		},
+		'../../../app/settings/server': {
+			settings: stubs.settings,
+		},
+	});
+
+const messages = [
+	{
+		_id: '65f46ce9e162d4ca5e6aba5c',
+		t: 'uj',
+		rid: 'GENERAL',
+		ts: new Date('2024-03-15T15:44:41.889Z'),
+		msg: 'matheus.barbosa',
+		u: {
+			_id: 'oNfuDGerMEiPoF7tq',
+			username: 'matheus.barbosa',
+			name: 'Matheus Barbosa Silva',
+		},
+		groupable: false,
+	},
+	{
+		_id: 'LmHvpSLmZfwcuNG9e',
+		rid: '65f7aa32d187cb7fe6779da3',
+		ts: new Date('2024-03-20T13:46:01.623Z'),
```

Comment:
Is there a specific reason we're using these exact dates instead of relative dates? This might give the impression of arbitrary date selection

Reply:
Not exactly, I just used an example response I had locally so that I didn't have to organize the data manually. I'm not sure about what would be the best approach, though :thinking:

If I had used random dates here, then I'd have to repeat the same process that is applied in the tested function to validate that the date is being correctly parsed to string. By using a fixed date, I don't need any magic and this can be checked as a string (as done [here](https://github.com/RocketChat/Rocket.Chat/pull/32046/files#diff-5d6db8c10f7d42233b9605b0caef5e44068dc875bd6c19eb33a8d4d3b1cdc23fR157) ). Which approach do you prefer?

Reply:
Got it! In this case, maybe we could add to a variable, so it can be a little more clear when reading the test. WDYT? Asking since I saw it's used multiple times - It's just a personal preference but not an issue at all. 

e.g: `messageSentDate`

Reply:
Done! I ended up using `faker`, which looks even better. So now I don't reference a specific date anymore :partying_face: 
Thanks again for the suggestions